msg/vehicle_status_flags.msg

```diff
@@ -24,6 +24,7 @@ bool position_reliant_on_optical_flow
 bool position_reliant_on_vision_position
 
 bool dead_reckoning
+bool condition_flight_terminated
```

Comment:
```suggestion
bool flight_terminated
```

We recently dropped the `condition_` prefix to minimize the total length of all concatenated fields (used by logger, etc).